src/aquarium-optimized/dawn/FishModelInstancedDrawDawn.cpp

```diff
@@ -175,6 +175,10 @@ void FishModelInstancedDrawDawn::init()
                                 {4, mNormalTexture->getTextureView()}});
     }
 
+    // mContextDawn->updateBufferData(mLightFactorBuffer,
+    //                            &mLightFactorUniforms, sizeof(LightFactorUniforms));
+    // mContextDawn->updateBufferData(mFishVertexBuffer,
+    //                            &mFishVertexUniforms, sizeof(FishVertexUniforms));

```

Comment:
Is this unfinished work?

Reply:
Thanks, I will remove the comments. setBufferData is implemented by buffer mapping sync method in Aquarium while updateBufferData is buffer mapping async method if  the toggle is enabled. And sync method is used in initialization and async method is used to update data for all constant buffers for every frame.